### ISSUE TITLE
Windows local source module bootstrapping

### ISSUE THREAD

**Issue by boro2g** [labd/mach-composer#133]
When running on windows, if you have the config:

```
components:
- name: ct-tf
  source: ../ct-tf
  version: 0.1.0
  integrations:  
  - commercetools
```
and run `mach apply`, this generates the `site.tf` with 

```
module "ct-tf" {
  source            = "C:\xxx\xxx\MachComposer\ct-tf"
```

which raises the exception:

```
Generated file deployments\main\mach-composer-test\site.tf
╷
│ Error: Invalid escape sequence
│
│   on site.tf line 99, in module "ct-tf":
│   99:   source            = "C:\xxx\xxx\MachComposer\ct-tf"
│
│ The symbol "x" is not a valid escape sequence selector.
```

If I manually edit `site.tf` and adjust the source path to be e.g. 
```
source            = "C:\\xxx\\xxx\\MachComposer\\ct-tf"
```

Then it can be installed ok using normal `terraform apply`.

**Comment by tleguijt** [comment]
Hi @boro2g what happens if you replace the backslashes with forward slashes?

**Comment by boro2g** [comment]
Sure, do you have instructions for how to run it off a different branch?

**Comment by mvantellingen** [comment]
Will be fixed in the new version (2.4.0), to be released asap
